General-Microproject-Information.md

```diff
@@ -50,8 +50,8 @@ and the implementation of that design were all reviewed and discussed,
 and after several iterations an improved version of the patch was
 accepted into our codebase. As a GSoC contributor, or Outreachy intern,
 you will be playing the role of the developer and engaging in a
-similar discussion. Get familar with the flow, need for clarity on
-both sides (i.e. you need to clearly explain your design, and need to
+similar discussion. Get familiar with the flow, need for clarity on
+both sides (i.e. you need to clearly defend your design, and need to
```

Comment:
The conflict with the master branch was likely that "defend" has been changed to "explain"  in 89a6e71888e96834bb15aa634f1e431c9b0f36ed, but when you rebased you fixed the conflict by putting back "defend" while I think you should have kept "explain".

Let me know if you want me to fix this.

Reply:
Except for this issue with the rebase the rest looks like great fixes. Thanks!

Reply:
My bad, thanks!